packages/fxa-auth-server/lib/error.js

```diff
@@ -283,6 +283,11 @@ AppError.translate = function (request, response) {
       info: payload.info,
       stack: response.stack,
     });
+
+    if (response.data) {
+      error.output.payload.data = JSON.stringify(response.data);
+    }
```

Comment:
![Screen Shot 2021-11-08 at 2 19 56 PM](https://user-images.githubusercontent.com/17437436/140804452-ffe28888-2886-4b2a-8f44-e5b6936710cd.png)

I confirmed locally that this logs a stringified version of `PayPalClientError.data` under Payload > data in Sentry when pointing `sentryDsn` to Ben's local test Sentry project in fxa-auth-server config.

Reply:
I'm not going to block on it, but ideally there'd be a test in `fxa-auth-server/test/local/error.js` for this new behavior. You could use the `do_reference_transaction_failure.json` fixture for it.

Reply:
test added